src/components/graph/graph.renderer.jsx

```diff
@@ -81,6 +90,43 @@ function _buildNodes(nodes, nodeCallbacks, config, highlightedNode, highlightedL
     });
 }
 
+/**
+ * Builds graph defs (for now markers, but we could also have gradients for instance).
+ * NOTE: defs are static svg graphical objects, thus we only need to render them once, the result
+ * is cached on the 1st call and from there we simply return the cached jsx.
+ * @returns {Object} graph reusable objects [defs](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/defs).
```

Comment:
Wrong jsdoc this `@returns {Function}` and that function is `_buildDefs` thus this should be documented on `_buildDefs`